tests/test_apps/run_dir_test/my_app.py

```diff
@@ -14,6 +14,10 @@ def my_app(_: DictConfig) -> None:
     run_dir = str(Path.cwd().relative_to(get_original_cwd()))
     time.sleep(2)
     run_dir_after_sleep = HydraConfig.get().run.dir
+    print(run_dir)
+    print(type(run_dir))
+    print(run_dir_after_sleep)
+    print(type(run_dir_after_sleep))
```

Comment:
This is very slow and expensive way to debug this.
You can use CircleCI "run with ssh" and ssh into an instance to debug.

Reply:
And it didnt even print anything :P